alicloud/table_alicloud_slb_load_balancer.go

```diff
@@ -0,0 +1,336 @@
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the resource group.",
+			},
+			{
+				Name:        "address_ip_version",
+				Type:        proto.ColumnType_STRING,
+				Description: "The IP version. Valid values: ipv4 and ipv6.",
+				Transform: transform.FromField("AddressIPVersion"),
+			},
+			{
+				Name:        "modification_protection_status",
+				Type:        proto.ColumnType_STRING,
+				Description: "Indicates whether the configuration read-only mode is enabled for the CLB instance. ",
+			},
+			{
+				Name:        "modification_protection_reason",
+				Type:        proto.ColumnType_STRING,
+				Description: "The reason why the configuration read-only mode is enabled.",
+			},
+			{
+				Name:        "bandwidth",
+				Type:        proto.ColumnType_INT,
+				Description: "The maximum bandwidth of the listener. Unit: Mbit/s.",
+			},
+			{
+				Name:        "internet_charge_type_alias",
+				Type:        proto.ColumnType_STRING,
+				Description: "The metering method of Internet data transfer. Valid values: paybybandwidth|paybytraffic.",
+			},
+			{
+				Name:        "load_balancer_spec",
+				Type:        proto.ColumnType_STRING,
+				Description: "The specification of the CLB instance.",
+			},
+			{
+				Name:        "delete_protection",
+				Type:        proto.ColumnType_STRING,
+				Description: "Indicates whether deletion protection is enabled for the CLB instance.",
+			},
+			{
+				Name:        "tags_src",
+				Type:        proto.ColumnType_JSON,
+				Description: "A list of tags.",
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Type:        proto.ColumnType_STRING,
+				Description: ColumnDescriptionTitle,
+				Transform:   transform.FromField("LoadBalancerName"),
+			},
+			{
+				Name:        "tags",
+				Type:        proto.ColumnType_STRING,
+				Description: ColumnDescriptionTags,
+				Transform:   transform.From(slbLoadbalancerTagMap),
+			},
+
+			// Alicloud standard columns
+			{
+				Name:        "region",
+				Description: ColumnDescriptionRegion,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("RegionId"),
+			},
+			{
+				Name:        "account_id",
+				Description: ColumnDescriptionAccount,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getCommonColumns,
+				Transform:   transform.FromField("AccountID"),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listSlbLoadBalancers(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create service connection
+	client, err := SLBService(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("alicloud_slb_load_balancer.listSlbLoadBalancers", "connection_error", err)
+		return nil, err
+	}
+
+	request := slb.CreateDescribeLoadBalancersRequest()
+	request.Scheme = "https"
+	request.PageSize = requests.NewInteger(50)
+	request.PageNumber = requests.NewInteger(1)
```

Comment:
@ParthaI can we add the standard limit check here?

Reply:
We are using same format as other tables are using.

Reply:
But why haven't we added the limit check in other tables then? I think this might be a good place to start.